packages/rs-drive-abci/src/abci/handlers.rs

```diff
@@ -136,7 +133,7 @@ where
 
         let app_hash = hex::encode(&response.app_hash);
 
-        tracing::info!(method = "init_chain", app_hash, "init chain executed");
+        tracing::info!(app_hash, "init chain executed");
```

Comment:
The same

Reply:
This adds app hash, and it is in "info" level. rs-tenderdash-abci logs request/response on "trace" level. I believe it's worth keeping it here for debugging.

Reply:
Agree!